Database/Tables/dbo.TreatmentBMP.sql

```diff
@@ -20,6 +20,7 @@ CREATE TABLE [dbo].[TreatmentBMP](
 	[RequiredFieldVisitsPerYear] [int] NULL,
 	[RequiredPostStormFieldVisitsPerYear] [int] NULL,
 	[InventoryIsVerified] [bit] NOT NULL,
+	[InventoryIsVerifiedDate] [datetime] NULL,
```

Comment:
I think a more descriptive name for this column would be "DateOfLastInventoryVerification"